--- src/net/common/ircthread.cpp.dir	2012-01-20 04:43:54.000000000 +0100
+++ src/net/common/ircthread.cpp	2012-01-20 04:44:08.000000000 +0100
@@ -19,7 +19,7 @@
 #include <net/socket_helper.h>
 #include <net/ircthread.h>
 #include <net/socket_msg.h>
-#include <libircclient/libircclient.h>
+#include <libircclient.h>
 #include <boost/algorithm/string/predicate.hpp>
 #include <queue>
 #include <sstream>
